File: Answers/06_for/e04_displayDiagonally.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Give odd number between <5,11>: ");
        if (int.TryParse(Console.ReadLine(), out int n) && n % 2 == 1 && n >= 5 && n <= 11)
        {
            for (int i = 0; i < n; i++)
            {
                Console.WriteLine($"{new string('*', i)}I{new string('*', n - i - 1)}");
            }
        }
        else
        {
            Console.WriteLine("not good number");
        }
    }
}

File: Exercies/06_for/e04_displayDiagonally.py
n = int(input("Give odd number between <5,11>: "))

if n % 2 == 1 and 5 <= n <= 11:
    for i in range(n):
        row = '*' * i + 'I' + '*' * (n - i - 1)
        print(row)
else:
    print("not good number")
